packages/e2e-tests/specs/editor/various/preview.test.js

```diff
@@ -248,6 +248,70 @@ describe( 'Preview', () => {
 
 		await previewPage.close();
 	} );
+
+	it( 'should not revert title during a preview right after switching from published to draft', async () => {
```

Comment:
```suggestion
	it( 'should display the correct preview when switching between published and draft statuses', async () => {
```

This title wasn't very clear to me. It would be better to be less specific about the implementation details of the test (e.g. `title`) and more descriptive about the actual case under test (i.e. the need to ensure we have the correct preview showing when we switch between publish and draft status). 

Moreover you could also include a link to the Issue that this PR fixes for context.

Reply:
This has been updated. Thanks!